src/functions-reference/unbounded_continuous_distributions.Rmd

```diff
@@ -168,6 +168,23 @@ $-\infty$ for $y$ above 8.25; `log1m(Phi_approx(...))` is more robust in the
 tails.
 `r since("2.21")`
 
+<!-- R; std_normal_qf; (T x); -->
+\index{{\tt \bfseries std\_normal\_qf }!{\tt (T x): R}|hyperpage}
+
+`R` **`std_normal_qf`**`(T x)`<br>\newline
+Returns the value of the inverse standard normal cdf $\Phi^{-1}$ at the
+specified quantile `x`. The `std_normal_qf` is equivalent to the 
+`inv_Phi` function.
+`r since("2.31")`
+
+<!-- R; std_normal_log_qf; (T x); -->
+\index{{\tt \bfseries std\_normal\_log\_qf }!{\tt (T x): R}|hyperpage}
+
+`R` **`std_normal_log_qf`**`(T x)`<br>\newline
+Return the value of the inverse standard normal cdf $\Phi^{-1}$ evaluated
+at the log of the specified quantile `x`. 
```

Comment:
I think it would be good to include 

> This function is equivalent to `std_normal_qf(exp(x))`, but is more numerically stable

Or similar, to explain why this function exists or why you might use it